tenable/io/filters.py

```diff
@@ -94,6 +94,39 @@ def access_group_filters(self, normalize=True):
         return self._use_cache('access_groups',
             'access-groups/filters', normalize=normalize)
 
+    def access_group_filters_v2(self, normalize=True):
```

Comment:
30:12: C0103: Variable name "f" doesn't conform to snake_case naming style (invalid-name)
Line 59 and 62, assign to single variable and then return the value by a single return

Reply:
Above comment addressed in previous commit